pages/common/lb.md

```diff
@@ -0,0 +1,28 @@
+# lb
+
+> A blog management tool.
+> More information: <https://github.com/LukeSmithxyz/lb>.
+
+- Make a new draft:
+
+`./lb new`
```

Comment:
I can't find it in the style guide but I feel like we had a rule to write every command like it was installed on the system.
```diff
- ./lb
+ lb
```

Reply:
ok cool, but another rule is to be beginner friendly and literally nobody installs `lb`. the readme doesn't have install instructions, there are no packages in any distribution and i'm not even sure if it works properly when installed.

Reply:
@sbrl what do you think?

Reply:
Even the documentation shows it just being run from local directory.

I think something like
```diff
./
{{path/to/lb}} ...
```
would be better

Reply:
Did you mean to **add** `{{path/to/lb}}`? If so, I'm not a fan of it. I'm fine with the way it's now.

Reply:
> I think something like
> 
> ```diff
> + ./
> - {{path/to/lb}} ...
> ```
> 
> would be better

> Did you mean to **add** `{{path/to/lb}}`?

Because then you can just as well assume it is installed in the system. `lb` is supposed to be in the pwd. Some users might want to install it into their systems, so `lb` makes some kind of sense, but I very much don't like `{{path/to/lb}}`, because either it is in the `PWD` or the `PATH`, **not** in another directory, **ever**.